### ISSUE TITLE
Option to bookmark multiple tabs

### ISSUE THREAD

**Issue by tomtom** [eoger/tabcenter-redux#276]
tabcenter-redux doesn't have trees and groups, which I think is a good thing. Sometimes I wish though there would be a way to bookmark a selection of tabs in a bookmark folder similar to firefox's own "Add bookmark for all tabs ..." feature. Would it be possible to add this feature? IMHO such a feature would be a great replacement for groups since firefox also supports opening all bookmarks in a folder in tabs.

If not, would it be possible to add a feature to move selected tabs to a new window so that I could afterwards use firefox's own "Add bookmark for all tabs ..."?

Regards

**Comment by StephGbzh** [comment]
The only item I still miss from the native tab context menu is "Bookmark all tabs".
All others were progressively added in successive releases.
"Bookmark all tabs" was requested in #66 along with the Undo close tab button but only this last one made it to the release and the issue was then closed :(

As a workaround, I use the extension https://addons.mozilla.org/en-US/firefox/addon/bookmark-all-button/ 
I looked a bit at the code of this extension and it's really short and simple so maybe it could be reused here without too much hassle ?

I know there are other issues open to be able to get the real native Firefox tab context menu but that may take a while until Mozilla accepts to do something about it and an even longer time until they really code the API.

For now I'm happy with the workaround though, I don't need to bookmark all tabs very often, so the extra button is hidden in the Overflow menu, taking zero space on screen but still quickly accessible when I need it.

**Comment by tomtom** [comment]
I don't understand your comment since FF's native tab context menu already contains an item "Bookmark all tabs". It's not possible to bookmark selected tabs only.

**Comment by StephGbzh** [comment]
Well, since I use Tab Center Redux to manage my tabs, I don't see the point of the native tabs, they are just taking up space for nothing so I hide the whole native tab bar through CSS. That means I don't have the native tab context menu => no "Bookmark all tabs". That's why I use a dedicated extension.

But I would prefer this to be integrated in Tab Center context menu directly, that's what I suggested.

That would indeed not cover your whole use case though, since you want only *some* tabs to be bookmarked.

**Comment by eoger** [comment]
To be honest I'm mostly waiting on [Bug 1376251](https://bugzilla.mozilla.org/show_bug.cgi?id=1376251) to be implemented at this point.

**Comment by Keith94** [comment]
This is a duplicate of #281.

Firefox will soon be getting native support for bookmarking a selection of tabs, too: https://bugzilla.mozilla.org/show_bug.cgi?id=1458067
